changelog/12483.removal.md

```diff
@@ -0,0 +1,4 @@
+In Rasa Open Source 3.0, we shipped Markers as an experimental feature. Since then, we discovered
+new requirements that made us rethink our approach. We will ship an evolution of Markers as part
+of the upcoming release(s) of Rasa Pro. Please reach out if you were using Markers with your production
+assistant, we're keen to engage with you!
```

Comment:
@andreaklmn are you OK with this language? This fragment will be visible in Rasa Open Source's changelog